### PR TITLE
Remove e2e tests

### PR DESCRIPTION
**Pull request by comzeradd**
All of those tests have either been moved to the theme or the blocks have been retired (eg. S2C).

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [leda](https://www-dev.greenpeace.org/test-leda) | [admin](https://www-dev.greenpeace.org/test-leda/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-leda/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-leda) | [composer-local.json](https://github.com/greenpeace/planet4-test-leda/blob/main/composer-local.json)

:watch: 2024.11.12 08:37:13

File: tests/e2e/editor-selector.spec.js
Comment (mleray) [inline tests/e2e/editor-selector.spec.js:1]: I don't see this one in the theme

Comment (comzeradd) [reply tests/e2e/editor-selector.spec.js:1]: Ah yes, good catch. Not sure if we need it though, as we have a lot of tests that touch the editor anyway 🤔 What do you think?

Comment (mleray) [reply tests/e2e/editor-selector.spec.js:1]: Yeah you're right, it's got nothing special

**Review by mleray — APPROVED** [review]
Looks good to me 👍 🧹

**Pull request merged**